app/styles/app.scss

```diff
@@ -1,10 +1,18 @@
 // Fix for Roboto font path
 $roboto-font-path: '../assets/';
+$ember-power-select-border-color: #64A5ED;
+$ember-power-select-border-radius: 0;
 
+@import "ember-power-select";
 @import "normalize";
 @import "foo.scss";
 
 .placeholder-size {
   width: 200px;
   height: 200px;
 }
+
+//Width of the select drop down for power-select: soda new page;
+.ember-power-select-trigger {
```

Comment:
can you please pull the styles for this out into a separate style sheet and import it into this one?